File: XamarinStore.iOS/Views/SpinnerCell.cs
using System;
using MonoTouch.UIKit;

namespace XamarinStore
{
	public class SpinnerCell : CustomViewCell
	{
		public SpinnerCell (): base(CreateView())
		{
		}
		static UIActivityIndicatorView CreateView()
		{
			var indicator = new UIActivityIndicatorView (UIActivityIndicatorViewStyle.Gray);
			indicator.StartAnimating ();
			return indicator;
		}
	}
}

File: F#/XamarinStore.iOS/Views/SpinnerCell.fs
namespace XamarinStore

open System
open MonoTouch.UIKit

type SpinnerCell() =
    inherit CustomViewCell(SpinnerCell.CreateView())

    static member CreateView() =
        let indicator = new UIActivityIndicatorView (UIActivityIndicatorViewStyle.Gray)
        indicator.StartAnimating ()
        indicator
